src/uu/env/src/env.rs

```diff
@@ -278,3 +323,25 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
         Err(code) => code,
     }
 }
+
+#[cfg(test)]
```

Comment:
these tests don't seem to be run when running `cargo test` from the root. Are they run as part of github actions?

Reply:
Are you sure they are not executed?

Reply:
To verify, I have added an `assert!(false)` at the end of my test:

When I then run the tests from the root, I still get an exit code of 0.
```bash
> git diff 
diff --git a/src/uu/env/src/env.rs b/src/uu/env/src/env.rs
index 1e975379..11d6c76b 100644
--- a/src/uu/env/src/env.rs
+++ b/src/uu/env/src/env.rs
@@ -360,5 +360,6 @@ mod test {
             Some(vec!["awk".into(), "-f".into(), "test.awk".into()]),
             parse_split_args(&OsString::from("--split-args awk -f test.awk"))
         );
+        assert!(false);
     }
 }
> cargo test --quiet > /dev/null 2>&1
> echo $?
0

```